app/templates/direct-award/index.html

```diff
@@ -30,47 +30,41 @@ <h1 class="govuk-heading-xl">
     Your saved searches
   </h1>
 
-  <div class="grid">
-
-    {{ summary.heading("Searching", id="searching_table") }}
-    {% call(item) summary.list_table(open_projects,
-      caption="Searching",
-      empty_message="You have no saved searches",
-      field_headings=[
-        "Name",
-        "Saved"  ],
-      field_headings_visible=True
-    ) %}
-      {% call summary.row() %}
-        {{ summary.service_link(item.name, url_for('direct_award.view_project', framework_family=framework.family, project_id=item.id)) }}
-        {{ summary.text(item.createdAt | datetimeformat ) }}
-      {% endcall %}
-    {% endcall %}
-
-    {{ summary.heading("Results exported", id="search_ended_table") }}
-    {% call(item) summary.list_table(closed_projects,
-      caption="Search ended",
-      empty_message="You have no saved searches",
-      field_headings=[
-        "Name",
-        "Exported",
-        "Status"
+  {% if open_projects|length %}
+    {{ govukTable({
+      "caption": "Searching",
+      "captionClasses": "govuk-heading-l",
```

Comment:
This probably wants to be a `govuk-heading-m` if it's an `h2`? Same for the other `h2`s on this template

Reply:
I agree this looks better - was trying to step down from the `<h1>` which is `xl`, but I think the visual hierarchy is still clear (as well as the semantic one).